### PR TITLE
Update channels map indexing indicator

### PR DESCRIPTION
**Pull request by softsimon**
New look with empty Geo channels and Node has Geo data which can happen when
* We just started up so no channels indexed
* Node channels don't have any geo data

Screenshot 2023-03-21 at 23 14 42

### CODE REVIEW

File: frontend/src/app/lightning/nodes-channels-map/nodes-channels-map.component.ts [hunk 1]
```diff
@@ -206,20 +205,13 @@ export class NodesChannelsMap implements OnInit {
...
     let title: object;
     if (channels.length === 0 && !this.placeholder) {
-      this.chartOptions = null;
-      this.showIndexingInProgress = true;
       this.isLoading = false;
-      return;
-    }
-
-    // empty map fallback
-    if (channels.length === 0 && this.placeholder) {
```
Comment (nymkappa) [inline frontend/src/app/lightning/nodes-channels-map/nodes-channels-map.component.ts:216]: I think this condition is for the node preview component used by unfurler so it looks your change would change its behavior. So need to make sure it does not break it.

Comment (softsimon) [reply frontend/src/app/lightning/nodes-channels-map/nodes-channels-map.component.ts:216]: OK I believe I restored the preview "preview component" behavior

**Comment by nymkappa** [comment]
I also show the generic message if we only have one data point for the node stats historical chart.
And I've added the same fix on the `Fee distribution` chart

image

**Review by nymkappa — APPROVED** [review]
Tested ACK @ v2.5.0-dev [[f1f6c481](https://github.com/mempool/mempool/commit/f1f6c481)]

Screenshot 2023-03-22 at 14 40 19

image

**Pull request merged**